lib/libutils/ext/include/compiler.h

```diff
@@ -34,18 +34,27 @@
  * ease portability and make the code easier to read.
  */
 
+/* define only if not already defined in bionic/libc/include/sys/cdefs.h */
```

Comment:
I'd rather have something like this:
*Some of the defines below is known to sometimes cause conflicts when this file is included from xtest in normal world. It is assumed that the conflicting defines has the same meaning in that environment. Surrounding the troublesome defines with #ifndef should be enough.*

Reply:
Ok. Forced push a new commit.